src/module.c

```diff
@@ -11142,6 +11068,105 @@ int moduleUnload(sds name) {
     return C_OK;
 }
 
+int moduleConfigsWereSet(RedisModule *module) {
+    listIter li;
+    listNode *ln;
+    listRewind(module->module_configs, &li);
+    while ((ln = listNext(&li))) {
+        ModuleConfig *module_config = listNodeValue(ln);
+        if (!module_config->is_initialized) {
+            return 0;
```

Comment:
correct me if i'm wrong, this loop will never run more than one iteration, right?
i.e. we can tell by the first config if it was initialized or not.
so we traded a single boolean in a module context (many instances of these), to a boolean per config (many instances of these too).
not sure which one i like less...

Reply:
WDYT about a single boolean in the module struct itself?

Reply:
i'm shocked i didn't think of that option 8-).

i guess i was hung on thinking it should be put in a struct that's transient and not long-lived, but that's not a real concern.
if we need just one boolean per loaded modules, that's obviously it's place.